ui/tests/e2e/env.js

```diff
@@ -1,11 +1,45 @@
+const core = require('@actions/core');
+
 const MESHERY_SERVER_URL = process.env.MESHERY_SERVER_URL || 'http://localhost:9081';
 const REMOTE_PROVIDER_URL = process.env.REMOTE_PROVIDER_URL || 'http://localhost:9876';
```

Comment:
@MUzairS15 I'm not sure how this will work, unless the GITHUB_TOKEN has access to the meshery-cloud repo. It might be that we should be using the staging-playground.meshery.io or the staging-meshery.layer5.io remote providers. I'm not sure. // @aabidsofi19

Reply:
Yes using staging-meshery.layer5.io is ideal here. // @jerensl 

localhost:9876 will not work in the CI unless remote provider is cloned and runs within the CI.

Reply:
If we want to run tests locally for debugging purposes against dev provider server, we can set the appropriate values for REMOTE_PROVIDER_USER_EMAIL and REMOTE_PROVIDER_USER_EMAIL environment variables.